charts/vc-authn-oidc/values.yaml

```diff
@@ -49,6 +50,9 @@ controller:
       - expired
       - failed
       - abandoned
+  userVariableSubsitution: |- 
+    # This is a default placeholder Python file 
+    # Add your default content here
```

Comment:
Default "blank" file here

Reply:
Looks good. Do we want to have a commented-out example of a substitution?

Reply:
Oh yeah, there's the example file in the source here
https://github.com/bcgov/vc-authn-oidc/blob/main/docker/oidc-controller/config/user_variable_substitution_example.py
so could add a function from that commented out, or just a comment saying they could look at that file

Reply:
My vote goes to adding a function/example for that file commented-out to make it foolproof.

Reply:
Added commented out file